sklearn/ensemble/tests/test_stacking_classifier.py

```diff
@@ -0,0 +1,217 @@
+    X, y = make_multilabel_classification(n_classes=2, n_labels=1,
+                                          allow_unlabeled=False,
+                                          random_state=123)
+    clf = OneVsRestClassifier(SVC(kernel='linear'))
+    clf_meta = OneVsRestClassifier(SVC(kernel='linear'))
+
+    eclf = StackingClassifier(estimators=[('ovr1', clf)],
+                              meta_estimator=clf_meta,
+                              method='decision_function')
+
+    try:
+        eclf.fit(X, y)
+    except NotImplementedError:
+        return
+
+
+def test_cv():
+    """Test cross-validation option cv=1"""
+    clf1 = LogisticRegression(random_state=1)
+    clfm = GaussianNB()
+    eclf = StackingClassifier(estimators=[('lr', clf1)], cv=1,
+                              meta_estimator=clfm).fit(X, y)
+    s = clf1.fit(X, y).predict_proba(X)[:, 1:]
+    assert_array_equal(clfm.fit(s, y).predict(s), eclf.predict(X))
+    assert_array_equal(clfm.fit(s, y).predict_proba(s),
+                       eclf.predict_proba(X))
+
+
+def test_gridsearch():
+    """Check GridSearch support."""
+    clf1 = LogisticRegression(random_state=1)
+    clf2 = RandomForestClassifier(random_state=1)
+    clf3 = GaussianNB()
+    clf_meta = LogisticRegression(random_state=1)
+    eclf = StackingClassifier(estimators=[
+                ('lr', clf1), ('rf', clf2), ('nb', clf3)],
+                meta_estimator=clf_meta)
+
+    params = {'lr__C': [1.0, 100.0],
+              'method': ['predict', 'predict_proba', 'predict_log_proba'],
+              'meta_estimator__C': [10.0, 20.0]}
+
+    grid = GridSearchCV(estimator=eclf, param_grid=params, cv=5)
+
+    grid.fit(iris.data, iris.target)
+
+
+def test_parallel_predict():
+    """Check parallel backend of StackingClassifier on iris."""
+    clf1 = LogisticRegression(random_state=123)
+    clf2 = RandomForestClassifier(random_state=123)
+    clf3 = GaussianNB()
+    clfm = GaussianNB()
+
+    eclf1 = StackingClassifier(
+        estimators=[('lr', clf1), ('rf', clf2), ('nb', clf3)],
+        meta_estimator=clfm,
+        n_jobs=1).fit(X, y)
+    eclf2 = StackingClassifier(
+        estimators=[('lr', clf1), ('rf', clf2), ('nb', clf3)],
+        meta_estimator=clfm,
+        n_jobs=2).fit(X, y)
+    assert_array_equal(eclf1.predict(X), eclf2.predict(X))
+    assert_array_equal(eclf1.predict_proba(X), eclf2.predict_proba(X))
+
+    # cv=1 option
+    eclf1 = StackingClassifier(
+        estimators=[('lr', clf1), ('rf', clf2), ('nb', clf3)],
+        meta_estimator=clfm,
+        n_jobs=1, cv=1).fit(X, y)
+    eclf2 = StackingClassifier(
+        estimators=[('lr', clf1), ('rf', clf2), ('nb', clf3)],
+        meta_estimator=clfm,
+        n_jobs=2, cv=1).fit(X, y)
+    assert_array_equal(eclf1.predict(X), eclf2.predict(X))
+    assert_array_equal(eclf1.predict_proba(X), eclf2.predict_proba(X))
+
+
+def test_sample_weight():
+    """Tests sample_weight parameter of StackingClassifier"""
```

Comment:
with nosetests, docstrings in test functions make test transcripts harder to read. make this a comment instead.